test/specs/lib/factories-test.js

```diff
@@ -395,6 +396,18 @@ describe('factories', () => {
         value: 'a string',
         mapValueToProps: () => ({ undef: undefined, nil: null, zero: 0, empty: '' }),
       })
+
```

Comment:
I wasn't completely sure if this was a good place to put the tests, but 'key' and 'childKey' are two separate sections that seem to deal exclusively with explicitly provided object/element props, so it didn't feel right putting it into one of those.

Reply:
Let's keep all tests for a given option under that `describe()` block.  Ideally, we only `describe(key)` once.